src/command_modules/azure-cli-interactive/azclishell/gather_commands.py

```diff
@@ -11,6 +11,11 @@
 from azclishell.command_tree import CommandBranch, CommandHead
 from azclishell.util import get_window_dim
 
+try:
+    from sets import Set as set
```

Comment:
`set` was introduced in 2.6: https://docs.python.org/2/library/stdtypes.html#set. I don't think we commit to python 2 versions earlier than 2.7.